### PR TITLE
Clone EXM130 stu3 module for generating r4 patients

### PR DESCRIPTION
**Pull request by lukoktonos**
The module did not need any updates in order to generate patients for r4.

See related PR:

### CODE REVIEW

File: src/main/resources/modules/EXM130-r4.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
+    "name": "EXM130",
```
Comment (mgramigna) [inline src/main/resources/modules/EXM130-r4.json]:
```suggestion
    "name": "EXM130-R4",
```

**Comment by mgramigna** [comment]
NIT: The `remarks` for some of the states in the module have snippets of CQL from the STU3 version, we could update these to be the new CQL R4

**Comment by mgramigna** [comment]
LGTM Pending two small comments

**Review by mgramigna — CHANGES_REQUESTED** [review]

**Review by mgramigna — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
Tested (through https://github.com/projecttacoma/fhir-patient-generator/pull/4), looks good.

**Comment by okeefm** [comment]
Note: restored the branch until https://github.com/projecttacoma/fhir-patient-generator/pull/4 is updated to point to the `abacus` branch

**Pull request merged**